moxin-frontend/src/data/store.rs

```diff
@@ -69,6 +69,8 @@ pub struct Store {
 
     pub preferences: Preferences,
     pub downloaded_files_dir: String,
+
+    pub active_chat_file: Option<FileID>,
```

Comment:
Not saying we should address it in this PR, but noticing we have `current_chat` representing the current chat session, but the information there is incomplete. We are storing the filename in this `Chat` instance but it could be a natural place to also store the `FileId`.

Reply:
Good catch, I missed that.
I'll merge this as is, and rework it a bit on the delete model PR, I have to tinker around with those values so it'll be easier to test if I have this merged.